tests/foreman/ui/test_contentview.py

```diff
@@ -108,3 +108,40 @@ def test_no_blank_page_on_language_switch(session, target_sat, module_org):
     with target_sat.ui_session(user=user.login, password=user_password) as session:
         session.user.update(user.login, {'user.language': 'Français'})
         assert session.contentview_new.read_french_lang_cv()
+
+
+@pytest.mark.tier2
+def test_file_cv_display(session, target_sat, module_org, module_product):
+    """Content-> Files displays only the Content Views associated with that file
+
+    :id: 41719f2f-2170-4b26-b65f-2063a1eac7fb
+
+    :steps:
+        1. Create a File Repo, and publish content to it
+        2. Add it to a CV, and publish that CV
+        3. Create a seperate CV, and publish it as well.
+        4. Navigate to the Content -> File section of the UI
+
+    :expectedresults: The CV containing the file repo is displayed, and no other CVs.
+
+    :BZ: 2026701
+
+    :customerscenario: true
```

Comment:
Can we add Verifies field as well?
```suggestion
    :customerscenario: true
    
    :Verifies: SAT-17081
```

Make sure the Jira id `SAT-17081` is correct  :smile_cat: